schemas-generated/flatbuffers/opendatafabric.fbs

```diff
@@ -334,9 +334,14 @@ enum MqttQos: int32 {
   ExactlyOnce,
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// MqttTopicSubscription
+// https://github.com/kamu-data/open-data-fabric/blob/master/open-data-fabric.md#mqtttopicsubscription-schema
+////////////////////////////////////////////////////////////////////////////////
+
 table MqttTopicSubscription {
   path: string;
-  qos: MqttQos = null;
+  qos: MqttQos;
```

Comment:
Here's an interesting thing, actually this field is not required, but if we add logic to `get_composite_type()`, the `= null` sign appears in many places.

```python
def get_composite_type(sch, required):
    if sch.get('type') == 'array':
        ...
    elif 'enum' in sch:
        ...    
    elif '$ref' in sch:
        # Current
        # return sch['$ref'].split('/')[-1]

        ptyp = sch['$ref'].split('/')[-1]
        if not required:
            ptyp += " = null"
        return ptyp
    else:
        ...
```